_rules/element-lang-valid-de46e4.md

```diff
@@ -141,22 +186,52 @@ The rule applies to elements within the `body` of a webpage. `html` elements are
 
```

Comment:
Update description in the line of "there is no element with a `lang` attribute value which is a descendant of a `body` element".